lms/djangoapps/teams/search_indexes.py

```diff
@@ -43,7 +44,15 @@ def data(self):
 
         Returns serialized object with additional search fields.
         """
-        serialized_course_team = CourseTeamSerializer(self.course_team).data
+        # Django Rest Framework v3.1 requires that we pass the request to the serializer
+        # so it can construct hyperlinks.  To avoid changing the interface of this object,
+        # we retrieve the request from the request cache.
+        context = {
+            "request": get_request_or_stub()
```

Comment:
Production code should not be calling this function. Instead, the test should mock out the `get_request()` function call and return a stub.